src/main/kotlin/com/kroffle/knitting/HttpControllers.kt

```diff
@@ -7,6 +7,13 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import org.springframework.web.server.ResponseStatusException
 
+@RestController
+@RequestMapping("/ping")
```

Comment:
요기는 트레일링 슬래쉬 넣을 수 있나여?!

Reply:
```suggestion
@RequestMapping("/ping/")
```
이렇게 할 수 있냐는 질문일까요?
아니면 `/ping/`으로 요청할 수 있냐는 질문일까요?

Reply:
무슨 기준인지 잘 모르겠는데..
`/ping` 는 404가 뜨고,
`/ping/`는 응답이 잘 오고,
`//ping/` 이것도 응답이 잘 오네요 🤔

Reply:
넵넵 /ping/ 할 수 있냐고 여쭤본 거였어요!!

혹시 /ping으로 정의하면 /ping/ 도 되나요???

Reply:
위에 답변 달았습니다!

Reply:
저건 /ping으로 정의할 때의 결과인가요??? 아니면 /ping/ ?! 
텍스트로 소통하니 모호해지네여 ㅋㅋㅋ

Reply:
둘다 그러네요 ㅋㅋㅋ

Reply:
ㅋㅋㅋㅋㅋ 일단 초기 셋업이니 넘어가시죠!! 담에 더 알아볼ㄱㅔ요 ㅎㅎ 시도해주셔서 감사합니당

Reply:
그 아래 ping 함수 매핑할 때
```
  @RequestMapping(value = "/*")
```

이런식으로 하면 된다길래 해봤는데 이것도 안 되는군요..
담에 알아보죠.. ㅎㅎ